AuthenticationAndAuthorization/AuthenticationAndAuthorization/appsettings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DataBase": "Host=localhost;Database=postgres;Username=postgres;Password=password;MinPoolSize=5;MaxPoolSize=100;"
+  },
+  "JwtOptions": {
+    "Issuer": "https://joydipkanjilal.com/",
+    "Audience": "https://joydipkanjilal.com/",
```

Comment:
Use localhost and port. Because for local development your localhost will be both Issuare and Audience. Read about iss and aud on the internet